uk_results/migrations/0030_populate_postresult_post_election.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from collections import defaultdict
+
+from django.db import migrations, models
+
+
+def set_post_election_from_post(apps, schema_editor):
+    """
+    This is far from idea. Try to guess the PostExtraElection
+    that this PostResult relates to. This will have to be done by looking
+    and the related memberships and assuming they're correct (sometimes they
+    won't be, and that will have to be fixed manually later).
```

Comment:
Yes, by "manually" I meant by looking at the results against the source documents etc. There aren't so many this year that this is a problem, and the data only spans 2 years.